django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
I think the exception silencing risk is less than changing the return value of `is_usable()` at this point in the release cycle. A user interacts with the database using Django with more than just `makemigrations` so any misconfigurations aren't likely to go unnoticed IMO.

@shaib (as reporter of [#25850](https://code.djangoproject.com/ticket/25850)), @aaugustin (as author of 25ce177e66fb0e88a39806f5a493e03f1801775a), any input?